libs/core/src/lib/renderer/registry.ts

```diff
@@ -37,7 +37,7 @@ export function getComponentClass(elementName: string): React.ReactType {
   const entry = elementMap.get(elementName) || elementMap.get(elementName.toLowerCase());
   if (!entry) {
     // throw new TypeError(`No known component for element ${elementName}.`);
-    return elementName;
+    return elementName as React.ReactType;
```

Comment:
Not exactly sure why I only started getting errors here just now (can't convert string to ReactType<any>) but this fixes it